docs_src/microservices/application/AdvancedTopics.md

```diff
@@ -424,3 +424,19 @@ If in secure mode, the secrets are retrieved from the secret store based on the
 
 If running in insecure mode, the secrets are retrieved from the *Writable.InsecureSecrets* configuration.
 
+### Registry Client
+
+**After initialization**, the configured registry client used by the SDK can be retrieved from the sdk instance at .RegistryClient.  This can then be used to look up host information for other services, or perform other operations supported by the registry.Client type in [go-mod-registry](https://github.com/edgexfoundry/go-mod-registry).  For example, to retrieve the URL for a given service:
+
```

Comment:
Add note about where to find the known EdgeX service keys which are in go-mode-core-contracts
https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/clients/constants.go#L58

notes have this syntax:
```
!!! note <note name>
         <note text>
```
Proper indent is required.

Reply:
Add another not about fact that RegistryClient could be nil. See my comment in Slack.

Reply:
Oops only saw the note in slack.  Just force pushed again :)